<ui version="4.0" >
 <class>SqlDelegateUi</class>
 <widget class="QWidget" name="SqlDelegateUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>48</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <property name="spacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLineEdit" name="lineEdit" >
       <property name="toolTip" >
        <string>Edit data here</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="nullButton" >
       <property name="toolTip" >
        <string>Set value to the NULL</string>
       </property>
       <property name="text" >
        <string>&amp;N</string>
       </property>
       <property name="shortcut" >
        <string>Ctrl+Shift+N</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="editButton" >
       <property name="toolTip" >
        <string>Open multiline editor</string>
       </property>
       <property name="text" >
        <string>&amp;E</string>
       </property>
       <property name="shortcut" >
        <string>Ctrl+Shift+E</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>nullButton</tabstop>
  <tabstop>editButton</tabstop>
 </tabstops>
 <resources>
  <include location="icons/icons.qrc" />
 </resources>
 <connections/>
</ui>
